envi/archs/thumb16/disasm.py

```diff
@@ -666,7 +666,7 @@ def render(self, mcanv, op, idx):
         fcond = cond_codes.get(cond)
         mcanv.addText("%s" % (fcond))
 
-    def getOperValue(self, idx, emu=None):
```

Comment:
old bugfix.  all ARM/Thumb Operands have to support this (due to ARM's tendency to update registers during instruction execution)

Reply:
all the other Operands do (for ARM), apparently this one slipped through and has been complaining for codeflow ever since.